frontend/device/kobo/device.lua

```diff
@@ -233,7 +233,12 @@ function Kobo:resume()
 end
 
 function Kobo:powerOff()
-    os.execute("poweroff")
+    local lfs = require("libs/libkoreader-lfs")
+    if lfs.attributes("/sbin/poweroff", "mode") == "file" then
+        os.execute("/sbin/poweroff")
```

Comment:
If there is such case in some platform it's better to setup the `PATH` variable in the launch script instead of adding ad-hoc logic in the code.